Formula/c/chrony.rb

```diff
@@ -20,13 +20,14 @@ class Chrony < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "52eb2a90d0a86c0259c2bf09ae9474262adb420c5bc6dd75ecb0ff8b5124f1c9"
   end
 
+  depends_on "gnutls"
   depends_on "nettle"
+  depends_on "pkg-config"
```

Comment:
Is `pkg-config` a runtime dependency?

Reply:
I believe it's only a build dependency. I just updated to add the `=> :build` directive. Does that seem correct?

Reply:
You'll need to move it above runtime dependencies but otherwise should be good.